code/modules/mob/living/simple_animal/hostile/commanded/guard_dog.dm

```diff
@@ -29,15 +29,16 @@
 	melee_damage_lower = 15
 	melee_damage_upper = 15
 
-	mob_size = 6
+	mob_size = 5
 
 	response_help = "pets"
 	response_harm = "hits"
 	response_disarm = "pushes"
 
 	hunger_enabled = 1 //so you can feed your dog or something
-	autoseek_food = 0
-	beg_for_food = 0
+	autoseek_food = 1
```

Comment:
I´d say thats a nogo for the guard dog (autoseek_food and beg_for_food)

Reply:
Any dog, even guard dog would seek food when they are starving. Maybe not beg

Reply:
I will remake it to be turned off by default. But turn on when it is starving